pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools>=38", "wheel", "numpy>=1.16.4", "cython>=0.29.12","scipy>=1.2.1"]
+requires = ["setuptools>=38", "wheel", "numpy>=1.16.4", "cython>=0.29.12","scipy>=1.2.1", "git+git://github.com/mohanwu/eigenpip.git"]
 build-backend = "setuptools.build_meta"
```

Comment:
Careful with the missing end-of-line :)

Reply:
Also, can you please add a `tox.ini` file to manage the tests?  This is preferable to `cd tests/; python -m unittest discover`, since it also checks the installation.

Resources: 

- Someone's [blog](https://christophergs.com/python/2020/04/12/python-tox-why-use-it-and-tutorial/).
- Tox [homepage](https://tox.wiki/en/latest/index.html).
- The `tox.ini` in pfjax.